packages/trader/src/AppV2/Components/TradeParameters/Duration/day.tsx

```diff
@@ -151,7 +152,8 @@ const DayInput = ({
                 textAlignment='center'
                 value={formatted_date}
                 disabled={duration_units_list.filter(item => item.value === 'd').length === 0}
-                onClick={() => {
+                onClick={(e: React.MouseEvent<HTMLInputElement, MouseEvent>) => {
+                    removeFocus(e);
```

Comment:
is there any chance to find solution which does not require to remember to remove focus from every single element? it feels like a recipe for sneaky bugs, I'm sure ppl will forget and it will sneak through the PRs. The same way missing dependencies in useHook sneaks through all the time and causes sneaky bugs all the time. Everyone knows about useEffect dependencies, but ppl still forget to add them and miss them in reviews, so basically its a source of bugs and unnecessary overhead in reviews.

cc @kate-deriv @nijil-deriv

Reply:
I will try to rework, @nijil-deriv gave me a tip to think about wrapper. The idea is to wrap all trade params with wrapper and inside that wrapper to track focus. If it'll be inside input, then remove it

Reply:
Based on comments I did another solution, could you please check?
[Link](https://github.com/deriv-com/deriv-app/pull/17219)
What was done:  `TradeParameters` (in packages/trader/src/AppV2/Components/TradeParameters/trade-parameters.tsx) is a container for all trade parameters, so inside of `useEffect` I added an event listener for `focusin`. If `event.target `contains specific custom attribute (`data-focus`), then I'll remove focus.
Custom attributes (`data-focus`) were added for all trade params inputs. That was the only way that works for me to indicate that it's correct input. We can't just remove focus from all inputs inside trade params components as user won't be able to even type something for stake, barrier and etc
@wojciech-deriv

Reply:
If alternative solution will melt your hearts, I'll merge that branch into a current one